commit 5209ace40194de81f5555757191b9821ed317602
Author: Ley Foon Tan <ley.foon.tan@intel.com>
Date:   Tue Feb 28 18:37:16 2017 +0800

    PCI: altera: Fix TLP_CFG_DW0 for TLP write
    
    commit 2a7275a3d867b228216886aae35e1f64291180b1 upstream.
    
    eb5767122feb ("PCI: altera: Simplify TLB_CFG_DW0 usage") used
    TLP_FMTTYPE_CFGRD* (instead of TLP_FMTTYPE_CFGWR*) for TLP writes, which
    causes writing to configuration space to fail.  Fix it by using correct
    FMTTYPE for write operation.
    
    Fixes: eb5767122feb ("PCI: altera: Simplify TLB_CFG_DW0 usage")
    Signed-off-by: Ley Foon Tan <ley.foon.tan@intel.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1292_5209ace40194de81f5555757191b9821ed317602.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1292_5209ace40194de81f5555757191b9821ed317602.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
